source/documentation/responsibility_model/responsibility_model.md

```diff
@@ -0,0 +1,70 @@
+# Responsibility model
+
+When you use the GOV.UK PaaS to host your app, both you and the GOV.UK PaaS have responsibilities. These responsibilities change depending on whether you deploy your app using a:
+
+- [standard](#standard-buildpack-responsibilities) buildpack
+- [custom](#custom-buildpack-responsibilities) buildpack
+- [Docker image](#docker-image-responsibilities)
+
+Regardless of which buildpack or Docker image you use, the GOV.UK PaaS team is always responsible for:
+
+- keeping the infrastructure your app runs on secure
+- providing secure app runtimes
```

Comment:
I think "secure app runtimes" is a bit misleading, as it sounds like "language runtime" which we use differently.

I'd be inclined to just drop the second point and say "GOV.UK PaaS team is always responsible for keeping the infrastructure your app runs on secure".